libraries/Microsoft.Bot.Configuration/BotConfiguration.cs

```diff
@@ -84,11 +84,22 @@ public static async Task<BotConfiguration> LoadFromFolderAsync(string folder, st
         }
 
         /// <summary>
-        /// load the configuration from a .bot file.
+        /// Load the bot configuration by looking in a folder and ing the first .bot file in the folder.
+        /// </summary>
+        /// <param name="folder">Folder to look for bot files. </param>
+        /// <param name="secret">Secret to use to encrypt keys. </param>
+        /// <returns>BotConfiguration.</returns>
+        public static BotConfiguration LoadFromFolder(string folder, string secret = null)
+        {
+            return BotConfiguration.LoadFromFolderAsync(folder, secret).GetAwaiter().GetResult();
+        }
+
+        /// <summary>
+        /// Load the configuration from a .bot file.
         /// </summary>
-        /// <param name="file">path to bot file</param>
-        /// <param name="secret">secret to use to decrypt the file on disk</param>
-        /// <returns>async task for BotConfiguration</returns>
+        /// <param name="file">Path to bot file. </param>
+        /// <param name="secret">Secret to use to decrypt the file on disk. </param>
+        /// <returns>Async task for BotConfiguration. </returns>
         public static async Task<BotConfiguration> LoadAsync(string file, string secret = null)
         {
```

Comment:
Null / empty / whitespace checks on file argument..

Reply:
Note: If the "file.OpenText" is doing all those checks already, please add a comment to that effect.